assets/css/less/wpbdp.less

```diff
@@ -1,138 +1,136 @@
-@import 'colors.less';
-@import 'common.less';
-@import 'wordpress-default-themes.less';
-@import 'datepicker.less';
+// Abstracts
+@variables-dir: "./wpbdp/abstracts/variables";
```

Comment:
Can you help me understand the "wpbdp" folder? Is this for front-end styles? Is there a more descriptive folder name we could use?

Reply:
Indeed, it involves front-end styles, and since we utilize `wpbdp.less` for such styles, I selected this name. I believe the front-end folder would be an appropriate choice.

Reply:
Is it possible to change it to "front-end"?

Reply:
That sounds like a good plan 👍